extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php

```diff
@@ -98,11 +98,10 @@ public static function onPageHistoryBeforeList() {
 
 	public static function onSendGridPostbackLogEvents( $events ) {
 		foreach ( $events as $event ) {
-			if ( $event[ 'event' ] == 'click' && strpos( $event[ 'category' ], self::EMAIL_CATEGORY ) !== false ) {
+			if ( self::checkIfAppreciationEmailEvent( $event ) ) {
```

Comment:
self::isAppreciationEmailEvent( $event ) would be shorter and more 'conventional'